src/views/Home.jsx

```diff
@@ -1,11 +1,34 @@
 import './Home.css';
+import { generateToken } from '@the-collab-lab/shopping-list-utils';
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { createNewList } from '../api/firebase';
+
+/** Home component that redirects a user to the List view if there is already a list created.
+ * If the user doesn't already have a list, a user can create one to be saved to Firestore and be redirected to the List view. */
+
+export function Home({ listToken, setListToken }) {
+	const navigate = useNavigate();
+
+	useEffect(() => {
+		if (listToken) {
+			navigate('/list');
+		}
+	}, []);
+
+	const createNewToken = async () => {
+		const newToken = generateToken();
+		setListToken(newToken);
+		createNewList(newToken);
```

Comment:
`createNewList(newToken)` should have `await` before it since it is an async function (`.then` is also an option as Matt points out above).

Reply:
Thank you, added!